keps/sig-auth/4940-psa-block-host-field-in-probes/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-auth
 participating-sigs:
   - sig-network
-status: provisional
```

Comment:
> enhancements/keps/prod-readiness/sig-auth/4940.yaml: no such file or directory

this didn't get marked as implementable, so never got a PRR review 🤦

will deal with that in a separate PR